api/Window.json

```diff
@@ -8441,19 +8441,19 @@
           "support": {
             "chrome": [
               {
-                "version_added": true
+                "version_added": "1"
               },
               {
-                "version_added": true,
+                "version_added": "1",
                 "alternative_name": "pageXOffset"
```

Comment:
This is api.Window.scrollX. I only noticed this now, but we also have a api.Window.pageXOffset entry, so these a `alternative_name` entries are weird, especially when both go back to ancient times. Can you clean that up in a separate PR?